### ISSUE TITLE
Hanging on Connecting to Client

### ISSUE THREAD

**Issue by ImmaZoni** [subspace/subspace-desktop#170]
Some users are starting to experience hanging on the client, they are getting stuck at the "Connecting to Client" screen and it doesn't seem to go past this.

Could be a symptom of the size of the network, could be something else going on.

**Comment by ozgunozerk** [comment]
Which version is this? 0.3.3?

**Comment by ImmaZoni** [comment]
v0.4.1

**Comment by ozgunozerk** [comment]
Can't reproduce this error. Do we know further details? When were these reports submitted?

**Comment by ozgunozerk** [comment]
Are you sure it is v0.4.1?

There was a bug in v0.4.1 that prevented the application starting directly from dashboard, and it was starting from scratch all the time. I just fixed it in the new pr (show-node-name), (and it was fixed on the multi-replica pr). 

So I don't think it is possible to get stuck on `connecting to client`, because the users will not be able to reach to `connecting to client` page in the first place... 
I believe this should be related to v0.4.0 or v0.3.3 (we had this connection issue in these versions). It got fixed in v0.4.1
@ImmaZoni
